src/modules/auth/resources/ResourcesListContainer.js

```diff
@@ -19,6 +19,9 @@ function ResourcesListContainer(props) {
 	const ref = useRef(null);
 	const { t } = useTranslation();
 
+	const deleteButtonResource = "authz:tenant:admin";
```

Comment:
@petrKavulok this resource is obsolete, please take a look to the xlsx file from purple sec and update the resources in affected containers accordingly

Reply:
Also, if not sure, please check it with Robin